src/server/utils/auth0Helpers.js

```diff
@@ -1,22 +1,18 @@
-import {AuthenticationClient, ManagementClient} from 'auth0';
+import {ManagementClient} from 'auth0';
 import getDotenv from '../../universal/utils/dotenv';
 
 // Import .env and expand variables:
 getDotenv();
 
-export const auth0AuthenticationClient = new AuthenticationClient({
-  domain: process.env.AUTH0_DOMAIN || 'parabol.auth0.com',
-  // eslint-disable-next-line max-len
-  token: process.env.AUTH0_MANAGEMENT_TOKEN || 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhdWQiOiJrRGZOTURJUXRxQ21lbVFGVHVoTjh6WVN4TlNFN3JjaSIsInNjb3BlcyI6eyJ1c2VycyI6eyJhY3Rpb25zIjpbInVwZGF0ZSJdfSwidXNlcnNfYXBwX21ldGFkYXRhIjp7ImFjdGlvbnMiOlsicmVhZCIsInVwZGF0ZSIsImRlbGV0ZSIsImNyZWF0ZSJdfX0sImlhdCI6MTQ3MTkxODAzNCwianRpIjoiMzdlOWFmYjRlMjEzOWYzMzNkMDkyNjY0NDBhMzk5MDEifQ.a6NANNe5IOzvFEomZm9OeeNpRfPjoMCUzA12PaK3zVg'
-});
+export const clientId = process.env.AUTH0_CLIENT_ID;
+export const clientSecret = process.env.AUTH0_CLIENT_SECRET;
+export const domain = process.env.AUTH0_DOMAIN;
+export const managementClientId = process.env.AUTH0_MANAGEMENT_CLIENT_ID;
+export const managementClientSecret = process.env.AUTH0_MANAGEMENT_CLIENT_SECRET;
+export const managementDomain = process.env.AUTH0_MANAGEMENT_DOMAIN;
 
 export const auth0ManagementClient = new ManagementClient({
-  domain: process.env.AUTH0_DOMAIN || 'parabol.auth0.com',
-  // eslint-disable-next-line max-len
-  token: process.env.AUTH0_MANAGEMENT_TOKEN || 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhdWQiOiJrRGZOTURJUXRxQ21lbVFGVHVoTjh6WVN4TlNFN3JjaSIsInNjb3BlcyI6eyJ1c2VycyI6eyJhY3Rpb25zIjpbInVwZGF0ZSJdfSwidXNlcnNfYXBwX21ldGFkYXRhIjp7ImFjdGlvbnMiOlsicmVhZCIsInVwZGF0ZSIsImRlbGV0ZSIsImNyZWF0ZSJdfX0sImlhdCI6MTQ3MTkxODAzNCwianRpIjoiMzdlOWFmYjRlMjEzOWYzMzNkMDkyNjY0NDBhMzk5MDEifQ.a6NANNe5IOzvFEomZm9OeeNpRfPjoMCUzA12PaK3zVg'
+  clientId: managementClientId,
+  clientSecret: managementClientSecret,
+  domain: managementDomain
```

Comment:
I've omitted `scope` from here. I tested that with the scope option omitted, it will receive the default scope from auth0 which is controlled in their dashboard. In our case, it's damned near everything (for now).